llarp/iwp/session.cpp

```diff
@@ -253,15 +256,22 @@ namespace llarp
             msg.SendACKS(util::memFn(&Session::EncryptAndSend, this), now);
           }
         }
+        std::priority_queue<
+            OutboundMessage*,
+            std::vector<OutboundMessage*>,
+            ComparePtr<OutboundMessage*>>
+            resend;
```

Comment:
I'd rather if this were called `resend_queue`, as `resend` feels like either a function or a boolean.